quartz.layout.ts

```diff
@@ -27,7 +27,7 @@ export const defaultContentPageLayout: PageLayout = {
     Component.MobileOnly(Component.Spacer()),
     Component.Search(),
     Component.Darkmode(),
-    Component.DesktopOnly(Component.Explorer()),
```

Comment:
We can still keep this i think. we can just say the behaviour in docs or somewhere.

Reply:
I have left the `DesktopOnly` out for now for testing. As soon as the code is approved, I'll update the documentation and restore the `DesktopOnly` before the PR is merged.

Reply:
actually we can just enable this fwiw